src/index.ts

```diff
@@ -1676,6 +1680,65 @@ class Spanner extends GrpcService {
   static pgJsonb(value): PGJsonb {
     return new codec.PGJsonb(value);
   }
+
+  /**
+   * @typedef IProtoMessageParams
+   * @property {object} value Proto Message Value. Serialized Buffer or message object.
```

Comment:
Could we be more specific about the type here instead of it being `object`?

Reply:
The object here would be message object generated by `protobufjs-cli`,  so I think we can't more specific here. Adding docs to make it more clear.